components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -67,13 +69,13 @@ be temporary, and you can try again later.</li>
     <string name="errorpage_title">Problem loading page</string>
     <string name="errorpage_refresh">Try Again</string>
 
-    <!-- NavigationDelegate.ERROR_CATEGORY_UNKNOWN -->
+    <!-- Category: NavigationDelegate.ERROR_CATEGORY_UNKNOWN -->
```

Comment:
This comment will only be shown for `error_generic_title`, and doesn't help there.

If the comment isn't vital for the actual code maintenance, I'd drop it.

Comments that are actually localizer-facing should be one per `<string>`.

Reply:
The comment was more for us to categorize sections for different error types. I'll remove it if it provides confusion.